README.md

```diff
@@ -0,0 +1,71 @@
+# Complete integration sample for the Corbado web component
+This is a sample implementation of frontend and backend where the Corbado web component is integrated. You can see the live demo [here](TODO: Link einfügen wenn online)
+
+**Note:** In this tutorial a customer system was built with some preexisting password-based users. Have a look at our [docs](TODO: Link einfügen) to see the changes you have to make if you don't have any users yet.
+
+## 1. File structure
+    .
+    ├── ...
+    ├── config                        
+    │   └── routes.yaml                 # Assigns paths to controller methods    
+    ├── docker                        
+    │   └── .env                        # Contains all docker environment variables   
+    ├── src                             
+    │   ├── Controller                  
+    │   │   ├── BackendController.php   # Manages endpoints for backend
+    │   └── └── FrontendController.php  # Manages endpoints for frontend
+    ├── templates                     
+    │   ├── home.html.twig              # Home page which you only get to see if you are logged in
+    │   └── login.html.twig             # Login page which contains the Corbado web component; Acts as landing page if you are not logged in
+    └── ...
+
+## 2. Setup
+
+### 2.1. Prerequisites
+
+Please follow the steps in our [setup guide](TODO: Link) to create and configure a project in our [developer panel](https://app.corabdo.com).
+
+### 2.2. Configure environment variables
+
+Use the values you obtained in step 2.1. to configure the following variables inside /docker/.env:
+1. **CNAME**: The individually generated CNAME.
+2. **PROJECT_ID**: The project ID.
+3. **API_SECRET**: The API secret.
+4. **NGROK_URL** Your individual ngrok URL (e.g. `https://eb70-212-204-96-162.eu.ngrok.io`)
+
+### 2.3. Start docker containers
+
+**Note:** Before continuing, please ensure you have [Docker](https://www.docker.com/products/docker-desktop/) installed and accessible from your shell.
+
+Use the following command to start the system:
+```
+docker compose up
+```
+**Note:** Please wait until all systems are ready. This can take some time if you have an older system.
```

Comment:
"containers" statt "systems" und "if you have an older system" löschen

Reply:
Wollte ausdrücken dass man nicht nur warten soll bis die container runnen sondern bis die container ihr Zeug wiederum initialisiert haben, aber ka obs das braucht